docs/rules.md

```diff
@@ -0,0 +1,130 @@
+
+## Keep all projects legal and appropriate...
+
+> Keep all projects legal and appropriate, and follow licensing requirements — for example, no piracy, malware or troll projects, and all code examples must avoid sexual and discriminatory language
+
+This rule is fairly self-explanitory, but we expect all of your projects to be both legal and appropriate. Although this isn't an exhaustive list, the following is unacceptable:
+
+* Projects that break the Minecraft EULA
+* Projects that enable piracy
+* Projects that contain malicious code, including code that targets specific players
+* Projects that make use of unlicensed (ARR) code from other projects, or otherwise break software licenses from other projects
+* Projects or code examples that contain sexualised or discriminatory language, including in variable names or folder/package structure
+* Projects that promote discrimination or real-life violence
+* Projects that don't otherwise violate this rule, but deliberately mask or hide their intentions (troll projects) — this doesn't include bugs, easter eggs or projects that exist to mess with players, as long as the projects are clear about their intentions
+
+In general, anything that would break another rule if used in casual conversation is not allowed in your code. We expect project maintainers to be professional, respectful and non-discriminatory in their code and interactions.
+
+## Follow the given topics and guidelines for each channel
+
+All of our channels exist for a specific reason. When chatting in any of the channel, your discussions should match the topic the channel was created for. It's pretty normal for conversations to drift off-topic after a while, but the people in that conversation are responsible for moving it to one of the general channels when that happens.
+
+To help you understand what each channel is for, please see our [channel guide](channels.md). You can also look at the description given for each channel on Discord.
+
+## Do not mention names from MCP, Mojang or other proprietary mappings...
+
+> Do not mention names from MCP, Mojang or other proprietary mappings, even if you use spoilers
+
+When writing code, you tend to give your variables, methods, classes and packages descriptive names — so that you know what they do at a glance. When Mojang released Minecraft, the included names were obfuscated (erased or masked), which makes it difficult to know what everything is for. Mappings exist to solve this problem, by taking Mojang's obfuscated names and translating them to something useful for mod developers.
+
+The Fabric project [is home to Yarn](https://github.com/FabricMC/yarn), a free, open-source alternative to proprietary Minecraft mapping projects. In order to ensure the legal security of Yarn's contributors, we require that proprietary names aren't talked about within our communities. This includes (but is not limited to) the following mappings:
```

Comment:
Consider limiting the link only to "to Yarn" or "Yarn"?

This is a suggestion hence why I'm not requesting changes.

Reply:
Links that span more words tend to be more noticeable, so I tend to err on the side of visibility.